test/lib/sync-fs.spec.js

```diff
@@ -1,6 +1,11 @@
-const assert = require('chai').assert;
-
+const chai = require('chai');
 const fs = require('../../src/lib/sync-fs');
+// NOTE: be careful with the names, this one is the original fs module
+// from nodeJS where the upper one is from our sync-fs module
+const fs_ = require('fs');
```

Comment:
Could we clear up a bit the confusion between these two imports? Maybe rename the first one to `syncFs` or rename the second one to `nodeFs`

I noticed that `sync-fs.js` re-exports node's `fs` module so we could also replace all `fs_` occurrences with `fs.fs` and drop the `const fs_ require('fs');` line

Either solution is perfectly fine, the goal is just to clear up any potential confusion with variable names

Reply:
Thanks for pointing this out. 
The pattern I've seen in the code is that when `sync-fs` is imported it's assigned to a variable named as `fs` as it is done here as well. However, for this test suite I need to mock the nod JS's `fs` which is also assigned to a variable named as `fs` but in this file `fs` name had already been taken so I assigned it to `fs_`. If I change the first `fs` variable then, the PR would have lots of changes unrelated to the issue in question and I wanted to keep it minimal and I named it as `fs_` because it would not in anyway mess up with the autocomplete of the IDE.

> I noticed that sync-fs.js re-exports node's fs module so we could also replace all fs_ occurrences with fs.fs and drop the const fs_ require('fs'); line

I am more aligned with this suggestion as it does not require changes for the previous code.
What do you think @m5r , @jkuester ?

Reply:
@sugat009 I think your rational makes sense!  Leave the first `fs` alone. As long as sinon works fine with it, just change `fs_` to `fs.fs`.  If sinon gives you trouble, just change `fs_` to `nodeFs`.  

(Also, Mokhtar is OOO next week.  I don't think we need to wait this PR on his feedback. So, I think we should proceed with your plan!)

Reply:
thanks @jkuester , updated.